statistics.md

```diff
@@ -236,6 +349,37 @@ When you analyze regular columns, you can use the `tidb_distsql_scan_concurrency
 
 When you analyze index columns, you can use the `tidb_index_serial_scan_concurrency` parameter to control the number of Region to be read at one time. The default value is `1`.
 
+### Persist ANALYZE configurations
+
+Since v5.4.0, TiDB supports persisting some `ANALYZE` configurations. With this feature, the existing configurations can be easily reused for future statistics collection.
+
+The following are the `ANALYZE` configurations that support persistence:
+
+| Configurations | Corresponding ANALYZE syntax |
+| --- | --- |
+| The number of histogram buckets | WITH NUM BUCKETS |
+| The number of Top-N  | WITH NUM TOPN |
+| The number of samples | WITH NUM SAMPLES |
+| The sampling rate | WITH FLOATNUM SAMPLERATE |
+| The `ANALYZE` column type | AnalyzeColumnOption ::= ( 'ALL COLUMNS' \| 'PREDICATE COLUMNS' \| 'COLUMNS' ColumnNameList ) |
+| The `ANALYZE` column | ColumnNameList ::= Identifier ( ',' Identifier )* |
+
+#### Enable ANALYZE configuration persistence
+
+The `ANALYZE` configuration persistence feature is enabled by default (the system variable `tidb_analyze_version` is `2` and `tidb_persist_analyze_options` is `ON` by default). You can use this feature to record the persistence configurations specified in the `ANALYZE` statement when executing the statement manually. Once recorded, the next time TiDB automatically updates statistics or you manually collect statistics without specifying these configuration, TiDB will collect statistics according to the recorded configurations.
+
+When you manually execute the `ANALYZE` statement multiple times with persistence configurations specified, TiDB overwrites the previously recorded persistent configuration using the new configurations specified by the latest `ANALYZE` statement.
+
+#### Disable ANALYZE configuration persistence
+
+To disable the `ANALYZE` configuration persistence feature, set the `tidb_persist_analyze_options` system variable to `OFF`. Because the `ANALYZE` configuration persistence feature is not applicable to `tidb_analyze_version = 1`, setting `tidb_analyze_version = 1` can turn off the feature too.
```

Comment:
```suggestion
To disable the `ANALYZE` configuration persistence feature, set the `tidb_persist_analyze_options` system variable to `OFF`. Because the `ANALYZE` configuration persistence feature is not applicable to `tidb_analyze_version = 1`, setting `tidb_analyze_version = 1` can also disable the feature.
```